src/modules/Game.class.js

```diff
@@ -6,63 +6,293 @@
  * Feel free to add more props and methods if needed.
  */
 class Game {
-  /**
-   * Creates a new game instance.
-   *
-   * @param {number[][]} initialState
-   * The initial state of the board.
-   * @default
-   * [[0, 0, 0, 0],
-   *  [0, 0, 0, 0],
-   *  [0, 0, 0, 0],
-   *  [0, 0, 0, 0]]
-   *
-   * If passed, the board will be initialized with the provided
-   * initial state.
-   */
-  constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+  static gameStatus = {
+    idle: 'idle',
+    playing: 'playing',
+    win: 'win',
+    lose: 'lose',
+  };
+
+  constructor(
+    initialState = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ],
+  ) {
+    this.status = Game.gameStatus.idle;
+    this.score = 0;
+    this.initialState = initialState;
+    this.state = initialState.map((row) => [...row]);
+  }
+
+  moveLeft() {
+    if (this.getStatus() !== Game.gameStatus.playing) {
+      return;
+    }
```

Comment:
The check for the game status is repeated in each move method. Consider creating a helper function like 'canMove' to encapsulate this check and reduce code duplication.